src/ralph/lib/mixins/fields.py

```diff
@@ -61,6 +61,67 @@ class NullableCharField(
     pass
 
 
+# NOTE(romcheg): NUMP stands for No Useles Migrations Please.
+#                It is recommended to refrain from checking the other meaning
+#                in UD.
+class NUMPFieldMixIn(object):
```

Comment:
what's the benefit of all this magic below? Why it's better that simple inheriting from `models.TextField` and overwriting `deconstruct` method?

Reply:
Well, that would be basically the same but less generic. Actually you pointed me to the solution that looks like this one. Later I and @ar4s discussed that it would be nice to have a convenient decorator to avoid subclassing for every specific case.